packages/fx-core/src/component/resource/botService/appStudio/appStudioClient.ts

```diff
@@ -61,26 +64,24 @@ export class AppStudioClient {
   ): Promise<IBotRegistration | undefined> {
     const axiosInstance = AppStudioClient.newAxiosInstance(token);
 
-    const getBotRegistrationResponse: AxiosResponse<any> | undefined = await RetryHandler.Retry(
-      async () => {
-        try {
-          return await axiosInstance.get(`${AppStudioClient.baseUrl}/api/botframework/${botId}`);
-        } catch (e) {
-          if (e.response?.status === 404) {
-            return e.response;
-          } else {
-            e.teamsfxUrlName = "<get-bot-registration>";
-            throw e;
-          }
+    return await RetryHandler.Retry(async () => {
+      try {
+        const response = await axiosInstance.get(
+          `${AppStudioClient.baseUrl}/api/botframework/${botId}`
+        );
+        return <IBotRegistration>response.data;
+      } catch (e) {
+        if (e.response?.status === 404) {
```

Comment:
Maybe you can use HttpStatusCode in component/constant/commonConstant.ts

Reply:
Agree - Will do.